### PR TITLE
Added instructions for discovery CLI

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: plugins/foreman_discovery/2.0/index.md [hunk 1]
```diff
@@ -210,7 +221,16 @@ To build a discovery image, please visit the foreman-discovery-image git
 [repository](https://github.com/theforeman/foreman-discovery-image) and find
 the README for further instructions.
 
-## 2.4 Upgrade
+## 2.4 Hammer CLI Foreman Discovery
+
+### 2.4.1 Installation
+
+Installation directions are the same as [hammer_cli_foreman plugin](https://github.com/theforeman/hammer-cli/blob/master/doc/installation.md),
+gem name: ```hammer_cli_foreman_discovery```.
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: Maybe we should refer to packages by default?

Comment (mmoll) [inline plugins/foreman_discovery/2.0/index.md]: yes please :package:

**Comment by orrabin** [comment]
@domcleal I added rpm to the installation part, it seems we don't have deb yet.

File: plugins/foreman_discovery/2.0/index.md [hunk 2]
```diff
@@ -464,6 +499,61 @@ When creating hostname patterns, make sure the resulting host names are
 good approach is to use unique information provided by facter (MAC address,
 BIOS or serial ID) or to randomize the hostname somehow.
 
+## 4.4 Hammer CLI Foreman Discovery plugin
+
+Confirm your setup by running $ hammer -h and check that the discovery command is listed.
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: use backticks (`) for commands

File: plugins/foreman_discovery/2.0/index.md [hunk 3]
```diff
@@ -351,7 +376,17 @@ click on *Refresh features* button and it will appear immediately.
 In *Infrastructure > Subnets* select the desired *Discovery Proxy* for each
 appropriate Subnet.
 
-## 3.3 Permissions
+## 3.3 Hammer CLI Foreman Discovery plugin
+
+The plugin must be configured in ```HAMMER_LOCATION/cli.modules.d/foreman_discovery.yml```, examples of [HAMMER_LOCATION](https://github.com/theforeman/hammer-cli/blob/master/doc/installation.md#locations)
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: Just to clear up this sentence:

``````
The plugin must be enabled in ```cli.modules.d/foreman_discovery.yml``` (see [Hammer config directories](http...)) as follows:
``````

File: plugins/foreman_discovery/2.0/index.md [hunk 4]
```diff
@@ -210,7 +221,21 @@ To build a discovery image, please visit the foreman-discovery-image git
 [repository](https://github.com/theforeman/foreman-discovery-image) and find
 the README for further instructions.
 
-## 2.4 Upgrade
+## 2.4 Hammer CLI Foreman Discovery
+
+### 2.4.1 Installation
+
+Before installing make sure you have installed [hammer_cli_foreman plugin](https://github.com/theforeman/hammer-cli/blob/master/doc/installation.md),
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: `have installed the [hammer....`

**Comment by domcleal** [comment]
Thanks, a couple of nitpicks but otherwise looks good.

**Comment by orrabin** [comment]
@domcleal I fixed your comments, thanks for the help with the wording.

**Comment by domcleal** [comment]
Thanks @orrabin!

**Pull request merged**